examples/flutter_timer/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_bloc: ^8.1.1
+  wave: ^0.2.2
```

Comment:
I intentionally removed this package because it was not maintained and appeared to cause lots of performance issues. I’d prefer to update the gif in the docs to match the example personally wdyt? Thanks again for the contribution!

Reply:
OK, thanks for the reply